test/controllers/welcome_controller_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'test_helper'
+
+class WelcomeControllerTest < ActionDispatch::IntegrationTest
+  test 'should get home' do
```

Comment:
👏 for the backend tests. It would be awesome, if you could also add tests for the frontend  part :)